react-frontend/src/components/DigitalFramework/digitalframeworktable.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react';
+import { Col, Row, Grid } from 'react-flexbox-grid';
+
+const DigitalFrameworkTable = (props) => {
+  const initial = window.innerWidth < 800;
+  const [isMobile, setIsMobile] = useState(initial);
+
+  useEffect(() => {
+    function handleResize() {
+      console.log(window.innerWidth);
+      if (window.innerWidth < 800) {
+        setIsMobile(true);
+      } else {
+        setIsMobile(false);
+      }
+    }
+
+    window.addEventListener('resize', handleResize);
+  });
+
+  //create list of rows using props
+  const entries = props.priorityActions.length;
+  //jsut use row col again but have title show up when screensize is less than a certain height
+  let body = [];
+
+  for (var i = 0; i < entries; i++) {
```

Comment:
These for loops seem a bit strange to me. We may want to rewrite as a map function.  Again, we can chat about that